metric_learn/constraints.py

```diff
@@ -59,7 +59,11 @@ def _pairs(self, num_constraints, same_label=True, max_iter=10):
     ab = np.array(list(ab)[:num_constraints], dtype=int)
     return self.known_label_idx[ab.T]
 
-  def chunks(self, num_chunks=100, chunk_size=2):
+  def chunks(self, num_chunks=100, chunk_size=2, random_state=None):
+    """
+    the random state object to be passed must be a random.seed() object and not a numpy random seed
+    """
+    state = random_state
```

Comment:
Ah, right. We should convert those calls to use the numpy random module. Then, the same technique used in the other methods will apply.